.circleci/scripts/install_npm.sh

```diff
@@ -7,6 +7,13 @@ cd "$(mktemp -d)"
 echo "Created test directory"
 npm init -y
 echo "Initialised new npm package"
+cd "$SCRIPT_DIR/../../installers/npm"
```

Comment:
__Nit__: is this going to resolve to an canonical path?

This might not matter based on how this is invoked, but there's a potential risk given we're using `cd` on line 6, *after* `$SCRIPT_DIR` is defined.

Reply:
Yes indeed, have made a change so we use `npm --prefix` instead which should make this easier to work with and stop us moving around so much